modules/hcp-ec2-client/main.tf

```diff
@@ -3,7 +3,7 @@ data "aws_ami" "ubuntu" {
 
   filter {
     name   = "name"
-    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+    values = ["ubuntu/images/hvm-ssd/ubuntu-hirsute-21.04-amd64-server-*"]
```

Comment:
we'll want to leave this unchanged. My understanding is that Focal has some very long-term support, [where other images don't.](https://wiki.ubuntu.com/Releases) We just had a mini-incident where another image version wasn't supported https://github.com/hashicorp/terraform-aws-hcp-consul/pull/48

Reply:
Yeh i remember that, this change was unwanted on my part.  will revert.